rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -1752,7 +1753,7 @@ namespace rsx
 		registers[NV406E_SET_CONTEXT_DMA_SEMAPHORE] = CELL_GCM_CONTEXT_DMA_SEMAPHORE_R;
 		registers[NV4097_SET_CONTEXT_DMA_SEMAPHORE] = CELL_GCM_CONTEXT_DMA_SEMAPHORE_RW;
 
-		if (get_current_renderer()->isHLE)
+		if (auto rsx = Emu.IsStopped() ? nullptr : get_current_renderer(); true)
```

Comment:
I think this entire thing can be moved to the if clause below, leaving a bare scope entry here